pyproject.toml

```diff
@@ -155,6 +155,7 @@ target-version = "py38"
 
 [tool.ruff.per-file-ignores]
 "tests/**" = ["D"]
+"tests/test_preprocess.py" = ["B018"]
```

Comment:
It's probably better to suppress this error in the specific file.

You can just add a line like the following after the imports:
`# ruff: noqa: B018`

And add a comment there to explain why it is being suppressed.

Alternatively, if this error messes with general test code, we can just add this to the previous line
```toml
"tests/**" = ["D", "B018"]
```

Reply:
Sounds good. I simply add `# ruff: noqa: B018` in that file and add some comments.